#Definition of the input variable type
#the name of the input variable
string name
#the value to be assigned
uint32 value